AdminWebsite/AdminWebsite/ClientApp/src/app/suitability/answer-list-entry/answer-list-entry.component.ts

```diff
@@ -22,6 +22,10 @@ export class AnswerListEntryComponent implements OnInit {
     this.expanded = !this.expanded;
   }
 
+  get isRepresentative(): boolean {
+    return this.questionnaire.hearingRole.toLowerCase().indexOf('solicitor') >= 0;
```

Comment:
hearing role may be undefined ?

Reply:
it is a mandatory field from the api

Reply:
however, can we avoid checking for solicitor here? it means that once we introduce another non-individual type, this will need to change.

I wonder, could we check if the `representee` field is set instead? so if you have any data there, we display it as `representing` otherwise as non-representing. What do you think?

Reply:
it will be exception, should not be, but will check anyway